README.md

```diff
@@ -269,7 +269,7 @@ buildscript {
   }
 
   dependencies {
-    classpath 'gradle.plugin.org.checkerframework:checkerframework-gradle-plugin:0.4.3-SNAPSHOT'
+    classpath 'gradle.plugin.org.checkerframework:checkerframework-gradle-plugin:0.4.4-SNAPSHOT'
```

Comment:
This file should include instructions for adding `checker-qual.jar` to the module path and as a required module if using the plugin with a module. See https://github.com/typetools/checker-framework/pull/2794/files#diff-f237d7d35cf0b39268f02bfae4298b0cR531-R546.

Reply:
I've added that text. Can you check it to make sure it's correct? Thanks